tsconfig.json

```diff
@@ -24,6 +23,8 @@
       "bazel-out/darwin_arm64-opt/bin",
       "bazel-out/darwin_arm64-fastbuild/bin"
     ]
+    // Note: we intentionally don't set "baseUrl": "." in order to force import
+    // module specifiers to be relative.
```

Comment:
Added this little note because I think I might've been the one who introduced `"baseUrl"` in the first place, thinking it was harmless - don't want others to repeat the same mistake :sweat_smile: